### PR TITLE
DockerCompose.yaml changes

### PR DESCRIPTION
**Pull request by isaac73**
# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] All new and existing tests passed.
- [x] The code follows the code style and conventions of this project.
- [ ] The change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

# Description of the change

Changes to yaml file after cache restructure

# Motivation for the change

...

### CODE REVIEW

File: scripts/docker/docker-compose.yml [hunk 1]
```diff
@@ -94,6 +94,8 @@ services:
     image: azureiotpcs/pcs-config-dotnet:1.0.0-preview
```
Comment (timlaverty) [inline scripts/docker/docker-compose.yml]: Why is this pointing to a preview container still?

File: scripts/docker/docker-compose.yml [hunk 2]
```diff
@@ -94,6 +94,8 @@ services:
...
     depends_on:
       - storageadapter
+      - telemetry
```
Comment (timlaverty) [inline scripts/docker/docker-compose.yml]: circular?

**Comment by dluc** [comment]
@isaac73 pls reply to comments and see if we can unblock the PR

**Comment by isaac73** [comment]
@timlaverty config depends on Device Simulation for seeding simulated devices and Telemetry for seeding rules. No, there isn't any circular dependency here.

Preview container has been updated to 1.0.0

**Pull request merged**